arangod/V8Server/v8-vocbase.cpp

```diff
@@ -1676,10 +1676,9 @@ static void JS_UseDatabase(v8::FunctionCallbackInfo<v8::Value> const& args) {
   void* orig = v8g->_vocbase;
   TRI_ASSERT(orig != nullptr);
 
-  v8g->_vocbase = vocbase;
-  static_cast<TRI_vocbase_t*>(orig)->release();
```

Comment:
I think this is still required. If we are not calling `release()` on `orig`, the reference count of `orig` will never be counted down again.

Reply:
Doesn't whatever produced by `WrapVocBase(isolate, vocbase)` decrease the count?